docs/src/componentDocs/Drawer/markdown/DrawerExamples.mdx

```diff
@@ -0,0 +1,14 @@
+import { BasicDrawer } from '../examples/BasicDrawer';
+import { ComplexDrawer } from '../examples/ComplexDrawer';
+
+# Basic Drawer
+
+A `<Drawer />` is a navigation menu that appears to the side of an application.
+
+<BasicDrawer />
+
+# Complex Drawer
+
+A `<Drawer />` can contain multiple `<DrawerNavGroup />` and nested `<DrawerNavItem />` with `activeItem` of Drawer.
```

Comment:
I am not sure why `activeItem` is mentioned in this sentence, so I removed it. But maybe you have a different intention here.

```suggestion
A `<Drawer />` can contain multiple `<DrawerNavGroup />` and nested `<DrawerNavItem />`.
```